test/upgrade/main.go

```diff
@@ -337,3 +370,96 @@ func createGameServerFile(agonesVersion string) string {
 
 	return gsPath
 }
+
+// Create a game server every ten seconds until the context is cancelled. The game server container
```

Comment:
Do you have a sense how many gameservers are created for each Agones version?

Reply:
It takes ~1 min to install Agones, so ~60 sec / 10 sec per version. We can certainly up that value (I had it at 5 sec on an earlier iteration).

Reply:
I put it back at create every 5 sec, and also added in wait time at each version to increase number game servers created.